docs/sources/reference/run.md

```diff
@@ -599,6 +600,19 @@ This means processes in container can be executed on cpu 1 and cpu 3.
 
 This means processes in container can be executed on cpu 0, cpu 1 and cpu 2.
 
+We can set mems in which to allow execution for containers. Only effective
+on NUMA systems.
+
+Examples:
+
+    $ docker run -ti --cpuset-mems="1,3" ubuntu:14.04 /bin/bash
+
+This means processes in container can be executed on memory node 1 and memory node 3.
```

Comment:
"This example restricts the processes in the container to only use memory from memory nodes 1 and 3."

Are the quotes surrounding "1,3" required or optional? Earlier examples didn't use quotes.